### PR TITLE
Move disconnectRepo function to its own file

### PR DESCRIPTION
**Pull request by alexweininger**
- Bump major package version because of breaking changes
- Move disconnectRepo function to its own file, to work towards removing commands out of tree items. 
- Kinda related to https://github.com/microsoft/vscode-azureappservice/issues/1277

### CODE REVIEW

File: appservice/src/disconnectRepo.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { SiteSourceControl } from 'azure-arm-website/lib/models';
+import { MessageItem } from 'vscode';
+import { IActionContext } from 'vscode-azureextensionui';
+import { editScmType } from './';
+import { ext } from './extensionVariables';
+import { localize } from './localize';
+import { ScmType } from './ScmType';
+import { DeploymentsTreeItem } from './tree/DeploymentsTreeItem';
+
+export async function disconnectRepo(context: IActionContext, node: DeploymentsTreeItem): Promise<void> {
```
Comment (EricJizbaMSFT) [inline appservice/src/disconnectRepo.ts:15]: I would make the `node` parameter less explicit. It should be the same type as the `node` parameter needed for `editScmType`

File: appservice/package-lock.json [hunk 2]
```diff
@@ -2639,4 +2639,4 @@
             }
         }
     }
-}
+}
```
[the remark this answers is not in the document]
Comment (EricJizbaMSFT) [reply appservice/package-lock.json:2642]: @alexweininger do you have to vscode-azuretools folder open in VS Code or just the "appservice" folder? If you open the "appservice" folder, it'll use these settings to keep consistent formatting: https://github.com/microsoft/vscode-azuretools/blob/master/appservice/.vscode/settings.json

[the remark this answers is not in the document]
Comment (alexweininger) [reply appservice/package-lock.json:2642]: 😭 lol why me

File: appservice/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.60.1",
+    "version": "0.61.0",
```
Comment (EricJizbaMSFT) [inline appservice/package.json:4]: I definitely like splitting this up into small chunks, but I prefer to consolidate breaking changes into one package update to some extent. So I would vote we merge this PR, but hold up on releasing until you get a few more changes in

File: appservice/package-lock.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (alexweininger) [reply appservice/package-lock.json:2642]: > @alexweininger do you have the "vscode-azuretools" folder open in VS Code or just the "appservice" folder? If you open the "appservice" folder, it'll use these settings to keep consistent formatting: https://github.com/microsoft/vscode-azuretools/blob/master/appservice/.vscode/settings.json

I have the "vscode-azuretools" folder added to my workspace. I added the "appservice" folder instead, and when I saved the file it formatted it correctly and added the newline. Thanks!

File: appservice/src/disconnectRepo.ts (hunk 1 above)
Comment (alexweininger) [reply appservice/src/disconnectRepo.ts]: Done

**Pull request merged**